backend/yonder/views.py

```diff
@@ -169,3 +173,33 @@ def put(self, request, *args, **kwargs):
     @swagger_auto_schema(tags=['comments'])
     def delete(self, request, *args, **kwargs):
         return self.destroy(request, *args, **kwargs)
+
+class inbox(generics.ListCreateAPIView):
```

Comment:
Since we are not using the List or Create here, this can be `generics.GenericAPIView` instead.